pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -34,8 +33,6 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:   mgrConfig.ResourceName,
 			Labels: c.getPodLabels(mgrConfig.DaemonID),
-			Annotations: map[string]string{"prometheus.io/scrape": "true",
```

Comment:
During upgrade will these annotations be lost since their yaml wasn't created originally with the annotations, right? Could we leave these annotations if none are specified in the crd? Then they could be overridden if specified.

Reply:
Done.